components/brave_rewards/resources/page/components/settingsPage.tsx

```diff
@@ -141,23 +141,27 @@ class SettingsPage extends React.Component<Props, State> {
     window.open('https://brave.com/privacy#rewards', '_blank')
   }
 
-  getGrantClaims = () => {
-    const { grants, ui } = this.props.rewardsData
+  getPromotionsClaims = () => {
+    const { promotions, ui } = this.props.rewardsData
 
-    if (!grants) {
+    if (!promotions) {
       return null
     }
 
+    let remainingPromotions = promotions.filter((promotion: Rewards.Promotion) => {
+      return promotion.status !== 4 // PromotionStatus::FINISHED
```

Comment:
nit: TODO so a const doesn't get forgotten

Reply:
right now we can't use direct definition so I added reference for a number